.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+## Change log
+- Add changes
+- Here
+
+## Other notes
+Additional notes if needed
+
+## Deployment reminder
+- [ ] PR requires deployment changes
+- [ ] Deployment PR done [link here]
+- [ ] Library updated if needed
+- [ ] PR merged
```

Comment:
This PR?

Reply:
No, pipeline PR.
This requires some re-wording 😄